app/views/catalog/_st_angular_form.html.haml

```diff
@@ -59,6 +59,17 @@
           %option{"value" => ""}
             = "<#{_('Unassigned')}>"
 
+    .form-group
+      %label.col-md-2.control-label{"for" => "catalog_id"}
+        = _('Would you like to remove associated resources as part of service retirement?')
```

Comment:
This seems odd as the only field text in the form of a question.  Maybe this should be: "Retirement: Remove associated resources" with options "Never, Pre-Playbook, Post-Playbook"

@tinaafitz @mkanoor @bzwei Thoughts?